### ISSUE TITLE
any way to automatically add braces for single line statement?

### ISSUE THREAD

**Issue by fyears** [yihui/formatR#56]
Any way to automatically add braces for single line code inside `if` or something like that?

For the source code like this:

```r
if (TRUE)
  a = 1
b = 2
```

Current default result for `tidy_source()`:

```r
if (TRUE) a = 1
b = 2
```

Expected formatted result (of the requested feature):

```r
if (TRUE) {
  a = 1
}
b = 2
```

**Comment by yihui** [comment]
Nope, but if you can figure out how to implement it, I'd be glad to take a look at your pull request :)
